docs/features/scripts.md

```diff
@@ -0,0 +1,50 @@
+---
+id: scripts
+title: Scripts support
+---
+
+### Working with a script
+
+Whenever, Metals opens a script file with `*.sc` extension, but only when it's
+not `*.worksheet.sc` or a build server is not already started for it, users will
+be prompted to choose whether they want to use [Ammonite](http://ammonite.io/)
+or [ScalaCli](https://scala-cli.virtuslab.org/) for using within the it.
+
+![scala-cli](https://i.imgur.com/ghR1Src.gif)
+
+Scripts are usually used for smaller programs and the main difference is that
+statements can be put on top level without a need for additional objects or
+methods.
+
+Once user have chosen a specific scripting tool, Metals will also suggest to
+import all newly opened scripts automatically or to always to do that manually.
+
+Both tools will start a build server in the background that will be able to
+compile your code and provide Metals with all the necessary information.
+
+ScalaCLI scripts can also be used standalone when ScalaCli generated `.bsp`
```

Comment:
I see both `ScalaCLI` and `ScalaCli`. Whatever you choose you should use the same throughout.